fedimint-server/src/net/peers.rs

```diff
@@ -85,6 +87,9 @@ struct PeerConnectionStateMachine<M> {
 
 struct CommonPeerConnectionState<M> {
     resend_queue: MessageQueue<M>,
```

Comment:
Should this be named "outgoing_queue"? It's not like it only contains items already sent/needing resent.

Reply:
Good catch! It got multiple functions now, both keeping un-ACKed messages and to-be-sent messages.